realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/AppConfigurationTests.kt

```diff
@@ -61,10 +62,6 @@ class AppConfigurationTests {
                 .authorizationHeaderName("CustomAuth")
                 .build()
         assertEquals("CustomAuth", config2.authorizationHeaderName)
-
-        // FIXME Add network check
-
-        // FIXME Add sync session check
```

Comment:
Are these checks integration tests on the configuration? Do we actually do want to do this here?

Reply:
I think the intent is to just see that they actually propagates down to the network layer. You could argue that is more of an integration test. Actually the tests are already in `SyncedRealmIntegrationTests.java` which I am in the process of migrating, so you can leave them out and I will include them in #6939